types/slate-react/index.d.ts

```diff
@@ -0,0 +1,108 @@
+// Type definitions for slate-react 0.12
+// Project: https://github.com/ianstormtaylor/slate
+// Definitions by: Andy Kent <https://github.com/andykent>
```

Comment:
Should these users be included here?

Reply:
Yes. Everyone in the credits are the original authors of these definitions, I'm just doing the work to put them into DefinitelyTyped.

Reply:
@paulvanbrenk Do you have an issue with the credits I've provided, or can this be merged in?